proj/src/main/webapp/notifications.js

```diff
@@ -105,12 +105,14 @@ function sendNotificationForChat(message) {
     const tag = sessionStorage.activeChatTag;
     const chatId = sessionStorage.activeChatId;
     const name = message.senderDisplay;
-    notificationBody = name+' has sent a message:\n'+message.content
+    notificationBody = name+' has sent a message:\n\n'+message.content
     let payload = {
         "to": getTopic(tag,chatId),
         "notification": {
             "title": tag,
-            "body": notificationBody
+            "body": notificationBody,
+            "icon": '/static/images/friendsChatting.png',
```

Comment:
Is there a reason you use the png instead of the svg? I think the svg is better because it will resize well.